collectors/charts.d.plugin/apcupsd/apcupsd.chart.sh

```diff
@@ -112,6 +112,9 @@ DIMENSION time time absolute 1 100
 CHART apcupsd_${host}.online '' "UPS ONLINE flag" "boolean" ups apcupsd.online line $((apcupsd_priority + 9)) $apcupsd_update_every '' '' 'apcupsd'
 DIMENSION online online absolute 0 1
 
+CHART apcupsd_${host}.selftest '' "UPS SELFTEST status" "Status_Code" ups apcupsd.selftest line $((apcupsd_priority + 10)) $apcupsd_update_every '' '' 'apcupsd'
```

Comment:
I think this should be a dimension per status, value is 0-1, 1 if the status is active.

Reply:
do you mean, you want one chart for selftest status = NG, and one chart for selftest status = BT?

Reply:
I mean the following:

- a chart a dimension for each status
- 1 value means that the status is active, 0 - inactive.

Reply:
okay then, not just for the bad status (NG, BT), i'll do that for all possible values.  

I'm guessing for another chart that I was about to submit, which reports a bad status for the APC status codea i.e. (0=ONLINE,1=ONBATT,2=OVERLOAD,3=LOWBATT,4=REPLACEBATT,5=NOBATT,6=SLAVE,7=SLAVEDOWN,8=COMMLOST,9=SHUTTING DOWN), #1-5 and #8 are bad and trigger 1 warning with the status code.  You would want me to split this into multiple charts being active/inactive too?

Reply:
I didn't mean bad/good status, I was saying about the values (numbers) - 1 active, 0 inactive. ONLINE, ONBATT, etc should be the chart dimensions (but make it lowercase). If the current status is online, then the online dimension value is 1, the rest - 0.